tests/integration-tests-base/pom.xml

```diff
@@ -72,4 +74,25 @@
     </plugins>
   </build>
 
+  <profiles>
```

Comment:
How does this work if you don't specify in the root pom.xml?

Can you show an example command?

Reply:
mvn test -DintegrationTests